Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,12 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def determine_dynamic_linker_path(formula)
+    if formula && ["glibc", "glibc25"].include?(formula.name)
```

Comment:
Both `binutils` and `gcc` are relocatable. In Linuxbrew we've manually marked them both as `cellar :any`, even though they're identified automatically as being no-relocatable. `binutil` is additionally portable because we compile it on CentOS 5. The `gcc` bottle depends on `glibc`, so it is relocatable but not portable.

It would be *fantastic* to make `glibc` relocatable. `glibc` itself does not support being moved to a new directory after it's configured and installed. We'd have to write that code. I'm sure it's possible, but not a small undertaking. The autoconf constant `@sysconfdir@` would have to be replaced with a function that determines at run-time where it's located in the file system, so that it can find its configuration files, libraries and stuff.